File: app/src/main/java/com/lzm/lightLive/http/Status.java
package com.lzm.lightLive.http;


/** Http状态*/
//todo 确认是否使用枚举？
public class Status {

    public static int LOADING = 800;

    /*Response 响应后结果*/
    public static int SUCCESS = 801;
    public static int FAILURE = 802;

    public static int ERROR = 803;

    public static String OK = "OK";
    public static String MSG_ERROR = "Unknown Error";
}

File: app/src/main/java/com/lzm/lightLive/http/Status.kt
package com.lzm.lightLive.http

/** Http状态 */ //todo 确认是否使用枚举？
object Status {
    var LOADING = 800

    /*Response 响应后结果*/
    var SUCCESS = 801
    var FAILURE = 802
    var ERROR = 803
    var OK = "OK"
    var MSG_ERROR = "Unknown Error"
}